packages/salesforcedx-sobjects-faux-generator/src/transformer/sobjectTransformer.ts

```diff
@@ -59,16 +58,16 @@ export class SObjectTransformer {
     this.result = { data: { cancelled: false } };
   }
 
-  public async transform(projectPath: string): Promise<SObjectRefreshResult> {
-    const projectFile = path.join(projectPath, SFDX_PROJECT_FILE);
```

Comment:
previously project path was getting passed around a lot unnecessarily.  I removed it where it made sense.